src/Apps/Artist/routes.tsx

```diff
@@ -122,8 +123,27 @@ export const routes: RouteConfig[] = [
     children: [
       // Routes in tabs
       {
-        path: "/",
+        path: ":regexParam(\\overview)?",
         getComponent: () => OverviewRoute,
+        render: ({ Component, props, match }) => {
```

Comment:
@zephraph - one other solution would be to do 

```
me { 
  id
}
```
in the query itself, as non-null indicates logged-in-ness

Reply:
That's true, yeah. 

I discovered this when looking through `buildClientApp`

https://github.com/artsy/reaction/blob/0535652b04b727f7149c6f24e6b1963097f30e1b/src/Artsy/Router/buildClientApp.tsx#L86

It is, so far as I can tell, driven by the system context. It's just passed through a different level. Is it worth adding more to get the user if it's already available otherwise?

Reply:
I _think_ that user is driven by headers, so its zero cost from a MP query perspective, and would be a dead-simple data-driven fix, versus leveraging router-specific stuff. That said i'm not opposed either way, we just need to be sure to pass it in on `buildServerApp` as well.

Reply:
It's already in `buildServerApp`

https://github.com/artsy/reaction/blob/0535652b04b727f7149c6f24e6b1963097f30e1b/src/Artsy/Router/buildServerApp.tsx#L82

I was able to trace it down to this commit: https://github.com/artsy/reaction/commit/36d07a381710d7ab7e0493b7f803ad935a6dc792

Reply:
I'm looking for other usages, but I'm not seeing anything. If that was the only thing that used it maybe we should just remove it and fallback to the query?

Reply:
Yeah, confirmed no other usages. Maybe we can just nix it and move to the query based approach.

Reply:
Its nice to query for what one wants :)

Reply:
Done